jenkins/ci.suse.de/templates/cloud-socok8s-integration.yaml

```diff
@@ -1,10 +1,32 @@
 - job-template:
-    name: '{name}-integration'
+    name: '{name}-integration-{tempest}'
     project-type: multibranch
-    periodic-folder-trigger: 5m
+    periodic-folder-trigger: '{trigger_frequency}'
```

Comment:
So this is only about how often to check for new branches or PR's. It's not how often to trigger.

Reply:
Agree. 
With recent change, added new daily job via pipeline project instead of using multibranch project-type.